src/components/core/styles/mixins/a11y.scss

```diff
@@ -7,6 +7,18 @@
   }
 }
 
+@mixin if-forced-colors-light {
```

Comment:
so far we did nowhere respect "prefers-color-scheme", so I'm not sure whether we should start this now?

Reply:
Because in high-contrast with white color scheme the high-contrast icons are invisible; the `filter: invert(1)` solves this issue.

<img width="1512" alt="Screenshot 2023-11-06 alle 11 02 53" src="https://github.com/lyne-design-system/lyne-components/assets/101575400/dc0b5a38-6449-443a-8d46-3be5fbe0eba6">

Reply:
I have the feeling that the icon itself should be inverted. like painted with black. but maybe it's the opposite problem then. and just applying color instead of filter, would that work?

Reply:
I have the feeling that the icon itself should be inverted. like painted with black. but maybe it's the opposite problem then. and just applying color instead of filter, would that work?

Reply:
I tried, no it does not